microcosm_daemon/runner.py

```diff
@@ -37,9 +38,9 @@ def __init__(self, processes, target, *args, **kwargs):
         self.kwargs = kwargs
 
     def run(self):
-        pool = Pool(processes=self.processes)
-        for _ in range(self.processes):
-            pool.apply_async(_start, (self.target,) + self.args, self.kwargs)
+        with closing(Pool(processes=self.processes)) as pool:
+            for _ in range(self.processes):
+                pool.apply_async(_start, (self.target,) + self.args, self.kwargs)
 
         pool.close()
```

Comment:
in theory the pool should be closed here, so I don't know what difference contextlib.closing will do